jbpm-task-assigning/kie-server-task-assigning-user-system-simple-kjar/pom.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-4.0.0.xsd">
+  
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.kie.server</groupId>
+    <artifactId>jbpm-task-assigning</artifactId>
+    <version>7.34.0-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>kie-server-task-assigning-user-system-simple-kjar</artifactId>
```

Comment:
Seems like a dead code (isn't part of Maven build).

Reply:
This is an example project that shows how the user system integration can be used with a kjar. This right now hangs from the jbpm-task-assigning module since it's the only place where all this development is located. But won't go into the kie-server-parent structure. Instead at some point a better location will be determined for this example. Don't know which at this moment, but we must see, probably optaplanner-examples-xxx or any other.